README.md

```diff
@@ -117,14 +117,47 @@ The Python integration tests make use of the VMM in varied configurations that
 aren’t overly complex and illustrate realistic use cases (e.g. one test runs a
 VMM with a virtio block device, one test will run a VMM with PCI, etc.).
 
-To be able to successfully run all the tests in this repo, there are a couple of
-resources that need to be found in `/tmp`. To generate the resources at the
-expected locations, you can run the following command:
+To be able to successfully run all the tests in this repo, pre-created
+resources are stored in S3. The resources are downloaded locally inside the
+`resources` directory. This is handled transparently by the test cases.
+Note: The resources once downloaded are cached, so they are not downloaded on
+every run.
+
+### Dependencies for Testing
```

Comment:
Can we move this section after the `Running Tests` section?

Reply:
I am just thinking whether this section is needed at all :-) Because the container image thing works just fine. If people indeed want to work outside the container, let the figure out.  It's more like an advanced usage, likely not required in the `README.md` file.